install_config/install/deploy_router.adoc

```diff
@@ -1113,74 +1113,87 @@ endif::[]
 
 Save this content to a file for use as the basis of your customized template.
 
-[[using-configmap-replace-template]]
 
+[[using-configmap-replace-template]]
 === Using a ConfigMap to Replace the Router Configuration Template
 
 A xref:../../dev_guide/configmaps.adoc#[configMap] permits you to customize
```

Comment:
the new xref format no longer allows for empty anchors, so be sure to add in the anchor found at the top of the page. In this case, dev-guide-configmaps.